todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksFragment.java

```diff
@@ -179,86 +138,77 @@ public boolean onMenuItemClick(MenuItem item) {
     }
 
     @Override
-    public void setLoadingIndicator(final boolean active) {
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
 
-        if (getView() == null) {
-            return;
-        }
-        final SwipeRefreshLayout srl =
-                (SwipeRefreshLayout) getView().findViewById(R.id.refresh_layout);
+        // Set up tasks view
+        ListView listView = mTasksFragBinding.tasksList;
+
+        // SnackBar setup
```

Comment:
could you wrap all the setups in methods, like you did in the `AddEditTaskFragment`, please?